CONTRIBUTING.md

```diff
@@ -22,4 +22,35 @@ We do all of NeMo-text-processing's development in the open. Contributions from
 11) Optional: if you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
 12) Send your PR and request a review
 
+# Notes for Language Contribution
+1) `en/graph_utils.py` is the de facto parent for all other `graph_util` functions. Please refrain from duplicating code logics into new `graph_utils.py` files and default to imports from `en/graph_utils.py` instead. `LANG/graph_utils.py` files should only contain new methods and variables. Not all new languages will require a submodule specific `graph_utils.py` file.
 
+2) When writing taggers for semiotic classes, you may need to tag additional token properties (e.g. grammatical gender, case) for accurate verbalization. For example, the Spanish ordinal `21.º` carries masculine gender and is verbalized with a specific spelling. As such, it would be desired for the TN tagger to tokenize the string with gender the gender property included:
+
+`"21.º" -> ordinal { integer: "vigésimo primero" gender: "masc" }`
+
+While the NeMo Text Processing library will permit any custom string in the tagger (provided proper parsing in the verbalizer), upstream Sparrowhawk support limits permissable strings, and will fail with custom token properties. As such, tagged properties should be limited to Sparrowhawk supported strings. 
+
+For all classes, Sparrowhawk support the `morphosyntactic_features` property, and it is recommended to default to this property for tagging additional features.
+
+`21.º" -> ordinal { integer: "vigésimo primero" morphosyntactic_features: "masc" }`
+
+For a list of all Sparrowhawk supported properties by class, see [here](https://github.com/yzhang123/sparrowhawk/blob/test/src/proto/semiotic_classes.proto)
+
+N.B. The same limitation applies for novel semiotic classes as well. Only predefined classes are supported in Sparrowhawk.
```

Comment:
definitely like this. would it make sense to add info on why we're restricting to sh supported classes / fields as an intro?

Reply:
Yeah that's a good call. Actually, do we have good info on why sparrowhawk deployment exists elsewhere in the documentation?